INSTALL.md

```diff
@@ -440,6 +440,13 @@ then this flag is optional and defaults to `ZLIB1` if not provided.
 This flag is optional and if not provided then `GNV$LIBZSHR`, `GNV$LIBZSHR32`
 or `GNV$LIBZSHR64` is used by default depending on the pointer size chosen.
 
+**RPM based distributions:** this option is not suggested to use, as internal
```

Comment:
This be better as part of or immediately below the _On Unix_ section above.

Reply:
IMO it should be somewhat more general. It is not just about RPM based distributions and the whole option comment should be clarified as it is misleading.

Reply:
Well i'd suggest to distinguish RPM as it provide dependency generator (auto Provides/Requires). This is a foundation for distribution RPM packaging policy i.e split development bits into separate subpackage

foo.src.rpm
libfoo1.rpm -> provides /usr/lib/libfoo.so.1 - (which is a real shared object)
libfoo1-devel.rpm -> provides /usr/include/libfoo.h and /usr/lib/libfoo.so - which is a symlink to real shared object library

After a second thought i may suggest this description:
```

**ATTENTION:** if you are going to use `zlib-dynamic` please make sure
your operating system or target operating systemd does have `libz.so`
in its standard installation set. Internal DSO of openssl uses `libz.so` to 
resolve real zlib shared object library. Please use `zlib` option instead if
you are going to distrbute openssl inside RPM or your distribution treat
'libz.so' as a development file and it is not a part of standard installation.
```